setup.py

```diff
@@ -63,4 +62,8 @@ def read(*rnames):
         'Django>=2.2',
         'pysaml2>=4.6.0',
         ],
+    tests_require=[
+        # Provides assert_called_once.
+        'mock;python_version < "3.6"',
```

Comment:
Since support for Python 3.5 was dropped, I believe the tests requirements are no longer necessary. Can also simplify `tox.ini` by removing the test requirements https://github.com/knaperek/djangosaml2/blob/master/tox.ini#L15.